baselines/fedmeta/README.md

```diff
@@ -0,0 +1,127 @@
+|   FedAvg(Meta)    |  SHAKESPEARE   |         4         |       400        |     10     |   Adam    |        0.001        | {'num_cpus': 4.0, 'num_gpus': 0.25 } |       -       |
+|   FedMeta(MAML)   |     FEMNST     |         4         |       2000       |     10     |   Adam    |   (0.001, 0.0001)   | {'num_cpus': 4.0, 'num_gpus': 1.0 }  |       5       |
+|   FedMeta(MAML)   |  SHAKESPEARE   |         4         |       400        |     10     |   Adam    |     (0.1, 0.01)     | {'num_cpus': 4.0, 'num_gpus': 1.0 }  |       1       |
+| FedMeta(Meta-SGD  |     FEMNST     |         4         |       2000       |     10     |   Adam    |   (0.001, 0.0001)   | {'num_cpus': 4.0, 'num_gpus': 1.0 }  |       5       |
+| FedMeta(Meta-SGD  |  SHAKESPEARE   |         4         |       400        |     10     |   Adam    |     (0.1, 0.01)     | {'num_cpus': 4.0, 'num_gpus': 1.0 }  |       1       |
+
+
+## Environment Setup
+```bash
+#Environment Setup
+Poetry install
+Poetry shell
+Pip install torch torch vision
+Pip install matplotlib
+pip install scikit-learn
+```
+
+## Running the Experiments
+
+****Download Dataset**** : Go [LEAF: A Benchmark for Federated Settings](https://github.com/TalwalkarLab/leaf) and Use the command below! You can download dataset (FEMNIST and SHAKESPEARE). 
+```bash
+#FEMNIST dataset Download command for these experiments
+./preprocess.sh -s niid --iu 1068 --sf 0.3 -k 0 -t sample
+
+#SHAKESEPEARE dataset Download command for these experiments
+./preprocess.sh -s niid --sf 0.16 -k 0 -t sample
+```
+
+****Start experiments**** : 
+```bash  
+# FedAvg + Femnist Dataset
+python main.py algo=fedavg data=femnist path=(your leaf dataset path)/leaf/data/shakespeare/data
+
+# FedAvg(Meta) + Femnist Dataset
+python main.py algo=fedavg_meta data=femnist path=../leaf/data/shakespeare/data
+
+# FedMeta(MAML) + Femnist Dataset
+python main.py algo=fedmeta_maml data=femnist path=../leaf/data/shakespeare/data
+
+# FedMeta(Meta-SGD) + Femnist Dataset
+python main.py algo=fedmeta_meta_sgd data=femnist path=../leaf/data/shakespeare/data
+
+
+
+#FedAvg + Shakespeare Dataset
+python main.py algo=fedavg data=shakespeare path=../leaf/data/shakespeare/data
+
+#FedAvg(Meta) + Shakespeare Dataset
+python main.py algo=fedavg_meta data=shakespeare path=../leaf/data/shakespeare/data
+
+#FedMeta(MAML) + Shakespeare Dataset
+python main.py algo=fedmeta_maml data=shakespeare path=../leaf/data/shakespeare/data
+
+#FedMeta(Meta-SGD) + Shakespeare Dataset
+python main.py algo=fedmeta_meta_sgd data=shakespeare path=../leaf/data/shakespeare/data
+
+```
+
+
+## Expected Results
+If you proceed with all of the above experiments, You can get a graph of your experiment results as shown below along that `./femnist or shakespeare/graph_params/result_graph.png`.
+
+**Femnist dataset experiment results**
+
+
+![](docs/femnist_result_graph.png)
```

Comment:
Could we have grid lines for this and the shakespeare plots? also, rename the directory to `_static`